src/SonarScanner.MSBuild.Shim/AdditionalFilesService.cs

```diff
@@ -132,6 +133,10 @@ private static string EnsureDot(string x)
         x = x.Trim();
         return x.StartsWith(".") ? x : $".{x}";
     }
+
+    private static bool ContainsBinOrObj(string path) =>
+        Array.Exists(path.Split(Path.DirectorySeparatorChar),
+            x => x.Equals("bin", StringComparison.OrdinalIgnoreCase) || x.Equals("obj", StringComparison.OrdinalIgnoreCase));
```

Comment:
I do not have a solution for this but we can't really tell if these are the bin or obj folders created by the compiler or if they are created by the user. 

There will be no way for the users to include these files (although this is probably a corner case)